tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_Contacts_RTL.py

```diff
@@ -0,0 +1,41 @@
+import pdb
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.contacts.app import Contacts
+from gaiatest.apps.contacts.regions.settings_form import SettingsForm
+from gaiatest.apps.contacts.regions.contact_form import ContactForm
+
+class TestContactsRTL(GaiaImageCompareTestCase):
+   
+    def test_contacts_app(self):
+
+        contacts = Contacts(self.marionette)
+        contacts.launch()
+        self.take_screenshot('main')
+        contacts.tap_settings()
+        self.take_screenshot('main')
+        settings_form = SettingsForm(self.marionette)
+        settings_form.tap_import_contacts()
+        self.take_screenshot('main')
+        settings_form.tap_import_from_sim()
+        self.take_screenshot('main')
+        settings_form.tap_back_from_import_contacts()
+        self.take_screenshot('main')
+        settings_form.tap_order_by_last_name()
+        self.take_screenshot('main')
+        settings_form.tap_done()
+        self.take_screenshot('main')
+        for i in range(0, 2):
+            GaiaImageCompareTestCase.scroll(self.marionette, 'down', contacts.screen_element.size['height'], screen=contacts.screen_element)
+            self.take_screenshot('main')
+
+        contacts.tap_new_contact()
+        self.take_screenshot('main')
+        contact_form = ContactForm(self.marionette)
+        contact_form.type_given_name(self.testvars['contact']['given_name'])
```

Comment:
Is this mean that if my testvar does not have these fields, the test would fail?  if so, it would be better to just create constatns can be substituted for each use of self.testvars